testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -31,8 +31,12 @@
 
 import itertools
 import warnings
+try:
```

Comment:
Thank you. It seems that six is more powerful than I thought after spending an hour playing with py3.io.StringIO.